lm75bd/lm75bd.c

```diff
@@ -27,7 +27,18 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
```

Comment:
always check if pointer arguments are null, in which case return error code invalid arg